### PR TITLE
[DEV-2721] Add rportd user command

### PR DESCRIPTION
**Pull request by domenipavec**
https://tracker.rport.io/issue/DEV-2721/manage-users-from-the-command-line

### CODE REVIEW

File: cmd/rportd/main.go [hunk 1]
```diff
@@ -243,59 +246,63 @@ var (
 func init() {
 	// Assign root cmd late to avoid initialization loop
 	RootCmd = &cobra.Command{
+		Use:     "rportd",
 		Version: chshare.BuildVersion,
 		Run:     runMain,
 	}
 
+	// lFlags are used only when running server
```
Comment (edoardoc) [inline cmd/rportd/main.go:254]: so, 
lFlags are used only for a running server, 
or else they are used to run a server (start a server)?

Comment (domenipavec) [reply cmd/rportd/main.go]: @edoardoc Changed the comment to `starting server`

File: cmd/rportd/user.go [hunk 2]
```diff
@@ -0,0 +1,175 @@
...
+	UserAddCommand    = "add"
+	UserDeleteCommand = "delete"
+	UserChangeCommand = "change"
+)
+
+var (
+	userCmd = &cobra.Command{
+		Use:   "user",
+		Short: "modify api users",
+		Long:  "Add, change or delete api users",
```
Comment (edoardoc) [inline cmd/rportd/user.go:28]: more as a curiosity: is there a difference between "user" and "api user"?

**Review by edoardoc — APPROVED** [review]

**Comment by domenipavec** [comment]
@KonradKuznicki PTAL

**Review by KonradKuznicki — APPROVED** [review]

**Pull request merged**